client/src/components/TrackItem.vue

```diff
@@ -321,7 +321,7 @@ export default defineComponent({
     }
   }
   .input-box {
-    border: 1px solid rgb(255, 255, 255, 0.15);
+    border: 1px solid rgba(255, 255, 255, 0.15);
```

Comment:
Seems unrelated?

Reply:
It was breaking build because the rollup sass plugin validates sass syntax.  Nothing in the webpack build was doing that.